src/query-performance-monitoring/query_performance_main.go

```diff
@@ -0,0 +1,57 @@
+package queryperformancemonitoring
+
+// this is the main go file for the query_monitoring package
+import (
```

Comment:
comment should be for the package, no?
```suggestion
// this is the main go file for the query_monitoring package
package queryperformancemonitoring

import (
```

Reply:
its just to mention that query performance monitoring starts from this file
not respective to the package